contracts/script/test/T1ERC7683PullE2E.s.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { Script } from "forge-std/Script.sol";
+import { console2 } from "forge-std/console2.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { TypeCasts } from "@hyperlane-xyz/libs/TypeCasts.sol";
+import { OrderData, OrderEncoder } from "intents-framework/libs/OrderEncoder.sol";
+import { OnchainCrossChainOrder } from "intents-framework/ERC7683/IERC7683.sol";
+import { T1ERC7683Pull } from "../../src/7683/T1ERC7683Pull.sol";
+import { T1Constants } from "../../src/libraries/constants/T1Constants.sol";
+
+uint32 constant DESTINATION_CHAIN = uint32(T1Constants.T1_DEVNET_CHAIN_ID);
+uint32 constant ORIGIN_CHAIN = uint32(T1Constants.L1_CHAIN_ID);
+
+// Step 1: Setup Alice's account, sign and relay intent
+contract AliceSetupScript is Script {
```

Comment:
I have trouble to execute it. I suggest to add a new section in the https://github.com/t1protocol/t1/blob/canary/contracts/TUTORIAL.md#interacting-with-%F0%9D%9A%9D%F0%9D%9F%B7s-erc-7683

Reply:
correct, this won't work until postman features are completed